platform/android/java/editor/src/horizonos/AndroidManifest.xml

```diff
@@ -37,6 +37,12 @@
     <!-- Temp removal of the 'REQUEST_INSTALL_PACKAGES' permission as it's currently forbidden by the Horizon OS store -->
     <uses-permission android:name="android.permission.REQUEST_INSTALL_PACKAGES" tools:node="remove" />
 
+    <!-- Passthrough feature -->
+    <uses-feature
+            android:name="android.hardware.camera2.any"
+            android:required="true"/>
+    <uses-permission android:name="horizonos.permission.HEADSET_CAMERA"/>
```

Comment:
Does the documentation state whether this is a runtime permission?

We'll need to update the Godot Meta Toolkit to allow users the ability to include the permission in their project.

Cc @dsnopek @devloglogan

Reply:
Permissions are documented here:
https://developers.meta.com/horizon/documentation/native/android/pca-native-documentation/#managing-permissions

Reply:
Calculation logic moved to CameraFeedAndroid.